smac/scenario/scenario.py

```diff
@@ -77,6 +77,10 @@ def __init__(self, scenario, cmd_args=None, run_id=1):
         self._groups = defaultdict(set)
         self._add_arguments()
 
+        # Make cutoff mandatory if run_obj is runtime
+        if 'run_obj' in scenario and scenario['run_obj'] == 'runtime':
```

Comment:
Can 'run_obj' be not in scenario? If the answer to this question is yes, we should move this check to another place.